.env.example

```diff
@@ -0,0 +1,5 @@
+APPLICATION_ID=""
+DISCORD_TOKEN=""
+PUBLIC_KEY=""
+GITHUB_SECRET=""
+DISCORD_WEBHOOK="" # make sure to append /github to the end of the webhook URL or else it will not work
```

Comment:
```suggestion
APPLICATION_ID=""
DISCORD_TOKEN=""
GITHUB_SECRET=""
PUBLIC_KEY=""
# make sure to append /github to the end of the URL on Discord webhooks
WEBHOOK_URL=""
```
alphabetical order (also that the used env var for the webhook seems to be `WEBHOOK_URL` and not `DISCORD_WEBHOOK`